packages/playground/src/utils/contracts.ts

```diff
@@ -99,11 +99,11 @@ export function formatConsumption(value: number): string {
   return normalizeBalance(value) + " TFT/hour";
 }
 
-export async function getNodeStatus(nodeIDs: (number | undefined)[]) {
+export async function getNodeInfo(nodeIDs: (number | undefined)[]) {
```

Comment:
how `getNodeInfo` but returns more than 1 nodeInfo
I think we can improve this part of code